python/tvm/contrib/pyfrontend.py

```diff
@@ -0,0 +1,305 @@
+"""Compiling a subset of Python to HalideIR"""
+import ast
+import inspect
+import types
+import operator
+#import numpy
+from .. import expr as _expr
+from .. import stmt as _stmt
+from .. import make as _make
+from .. import api  as _api
+from .. import ir_pass  as _ir_pass
+from .. import build_module as builder
+
+#pylint: disable=missing-docstring, invalid-name, consider-merging-isinstance, no-else-return
+#pylint: disable=inconsistent-return-statements, eval-used
+
+NOOP = _make.Evaluate(_api.const(0))
```

Comment:
We may use NOP, the same meaning with NOP in some hardware instruction sets, 
eg http://infocenter.arm.com/help/index.jsp?topic=/com.arm.doc.dui0489h/Cjafcggi.html.
```
NOP
No Operation.
Show/hideSyntax
NOP{cond}
```

Reply:
This is done.